jetty-slf4j-impl/src/main/java/org/eclipse/jetty/logging/StdErrAppender.java

```diff
@@ -0,0 +1,251 @@
+//
+// ========================================================================
+// Copyright (c) 1995-2020 Mort Bay Consulting Pty Ltd and others.
+//
+// This program and the accompanying materials are made available under
+// the terms of the Eclipse Public License 2.0 which is available at
+// https://www.eclipse.org/legal/epl-2.0
+//
+// This Source Code may also be made available under the following
+// Secondary Licenses when the conditions for such availability set
+// forth in the Eclipse Public License, v. 2.0 are satisfied:
+// the Apache License v2.0 which is available at
+// https://www.apache.org/licenses/LICENSE-2.0
+//
+// SPDX-License-Identifier: EPL-2.0 OR Apache-2.0
+// ========================================================================
+//
+
+package org.eclipse.jetty.logging;
+
+import java.io.PrintStream;
+import java.time.Instant;
+import java.time.ZoneId;
+import java.time.ZonedDateTime;
+import java.time.format.DateTimeFormatter;
+import java.util.Objects;
+import java.util.TimeZone;
+
+import org.slf4j.event.Level;
+import org.slf4j.helpers.FormattingTuple;
+import org.slf4j.helpers.MessageFormatter;
+
+public class StdErrAppender implements JettyAppender
+{
+    private final DateTimeFormatter timestampFormatter;
+    private final ZoneId timezone;
+    /**
+     * True to have output show condensed logger names, false to use the as defined long names.
+     */
+    private final boolean condensedNames;
+    /**
+     * True to have messages escaped for control characters, false to leave messages alone.
+     */
+    private final boolean escapedMessages;
+    /**
+     * True to have formatting be based on the strict definition of Slf4J's {@link MessageFormatter},
+     * where there has to be a match to the number of <code>{}</code> in the format string
```

Comment:
Use `{@code ...}` instead of `<code>`.

Reply:
Using `{@code ...}` for content that contains `{` or `}` has caused problems in the javadoc tooling (in the past).